DDDetectors/src/Mask_o1_v01_geo.cpp

```diff
@@ -92,15 +115,15 @@ static dd4hep::Ref_t create_element(dd4hep::Detector& description,
 	     << std::setw(35) << volName
 	     << std::setw(15) << sectionMat.name()
 	     << std::setw(8) << phi1
-	     << std::setw(8) << phi2;
+	     << std::setw(8) << phi2
+	     << std::setw(8) << ssensitive;
 
     printout(dd4hep::INFO, "DD4hep_Mask", pipeInfo.str() );
 
     // things which can be calculated immediately
     const double zHalf       = fabs(zEnd - zStart) * 0.5; // half z length of the cone
     const double zPosition   = fabs(zEnd + zStart) * 0.5; // middle z position
-    Material material    = sectionMat;
```

Comment:
This wasn't necessary, but can you leave it as is to reduce the amount of changes?